### ISSUE TITLE
Use custom user profile fields for accessibility.

### ISSUE THREAD

**Issue by gjb2048** [gjb2048/moodle-format_grid#23]
NTS: Conversation with Mags.

Custom user profile fields can have one of three visibility settings:

        Visible to everyone
        Not visible
        Visible to user

The Not visible setting would typically be set by an administrator who wants to hold private data on the users. The Visible to user setting would normally be selected for a field that holds sensitive information, while the Visible to everyone setting can be used for any type of information.

Therefore can use as a means of telling the Grid format to display as 'topics' format for a given user.  Then screen reader software (such as JAWS: http://www.freedomscientific.com/Downloads/JAWS) can describe the whole page.

**Comment by gjb2048** [comment]
Info on: https://docs.moodle.org/30/en/User_profile_fields

Looks like custom fields are added to the $USER global, i.e. 'accessibility' with:

````
global $USER;
error_log(print_r($USER->profile, true));
````

gives:

````
[17-Feb-2016 13:29:09 Europe/London] Array
(
    [accessibility] => 1
)
````

**Comment by gjb2048** [comment]
Ok, use 'accessible'.

**Comment by gjb2048** [comment]
Implemented in V3.0.0.4
